course/include/isometry.h

```diff
@@ -0,0 +1,84 @@
+#ifndef ISOMETRY_H
+#define ISOMETRY_H
+
+#include <cstdint>
+#include <iostream>
+#include <ostream>
+
+namespace ekumen {
+namespace math {
+
+class Vector3{
+    public:
+        // Constructors
+        Vector3(double x = 0, double y = 0, double z = 0) : x_{x}, y_{y}, z_{z} {};
```

Comment:
The semicolons at the end are not needed for inlined member functions. A linter would complain.